unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -106,20 +107,19 @@ def _trigger_complete(
                 temperature=temperature,
                 options=options,  # type: ignore
```

Comment:
same as above. Is options actually consumed by the API?

Reply:
see answer above